status-im/status-react
# Enable main wallet screen. WIP features are hidden behind flag

### Summary:

Enable main wallet screen in all builds.

### Steps to test:
- Verify Wallet tab is enabled
- Open Wallet tab
- All icons should be actionable but show `TODO` popup

status: ready
